app/templates/Users/view.php

```diff
@@ -0,0 +1,77 @@
+            <?= $this->Html->link(__('New User'), ['action' => 'add'], ['class' => 'side-nav-item']) ?>
+        </div>
+    </aside>
+    <div class="column-responsive column-80">
+        <div class="users view content">
+            <h3><?= h($user->email) ?></h3>
+            <table>
+                <tr>
+                    <th><?= __('Email') ?></th>
+                    <td><?= h($user->email) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Id') ?></th>
+                    <td><?= $this->Number->format($user->id) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Created') ?></th>
+                    <td><?= h($user->created) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Modified') ?></th>
+                    <td><?= h($user->modified) ?></td>
+                </tr>
+            </table>
+            <div class="related">
+                <h4><?= __('Related Articles') ?></h4>
+                <?php if (!empty($user->articles)) : ?>
+                <div class="table-responsive">
+                    <table>
+                        <tr>
+                            <th><?= __('Id') ?></th>
+                            <th><?= __('User Id') ?></th>
+                            <th><?= __('Title') ?></th>
+                            <th><?= __('Slug') ?></th>
+                            <th><?= __('Body') ?></th>
+                            <th><?= __('Published') ?></th>
+                            <th><?= __('Created') ?></th>
+                            <th><?= __('Modified') ?></th>
+                            <th class="actions"><?= __('Actions') ?></th>
+                        </tr>
+                        <?php foreach ($user->articles as $articles) : ?>
+                        <tr>
+                            <td><?= h($articles->id) ?></td>
+                            <td><?= h($articles->user_id) ?></td>
+                            <td><?= h($articles->title) ?></td>
+                            <td><?= h($articles->slug) ?></td>
+                            <td><?= h($articles->body) ?></td>
+                            <td><?= h($articles->published) ?></td>
+                            <td><?= h($articles->created) ?></td>
+                            <td><?= h($articles->modified) ?></td>
+                            <td class="actions">
+                                <?= $this->Html->link(__('View'), ['controller' => 'Articles', 'action' => 'view', $articles->id]) ?>
+                                <?= $this->Html->link(__('Edit'), ['controller' => 'Articles', 'action' => 'edit', $articles->id]) ?>
+                                <?= $this->Form->postLink(__('Delete'), ['controller' => 'Articles', 'action' => 'delete', $articles->id], ['confirm' => __('Are you sure you want to delete # {0}?', $articles->id)]) ?>
```

Comment:
I'm sorry. Similar issue as highlighted in https://github.com/sergio9977/tutorial-cakephp/pull/3#discussion_r1385020604

Please check.